5/README.md

```diff
@@ -0,0 +1,166 @@
+```
+shortname: 4/STANDARIDIZE-DC
+name: Standard process to set up a local node for development & testing, using Docker Compose
+type: standard
+status: raw
+editor: Muawia Khan <muawia@bigchaindb.com>
+```
+
+
+## Abstract
+In the current BigchainDB repository we have 8 docker-compose files, each serving a specific function:
+
+- docker-compose.yml
+  - DEPRECATED: Used for deployment of single node development BigchainDB v1.3.
+- docker-compose.travis.yml
+  - Used to setup the continous integration environment on Travis.
+- docker-compose.tendermint.yml
+  - Used to deploy Single node development environment for BigchainDB(integrated with Tendermint).
+- docker-compose.network.yml
+  - Used to deploy 4 node development environment for BigchainDB(integrated with Tendermint)
+- docker-compose.docs.yml
+  - Used to build and host BigchainDB docs.
+- docker-compose.ssl.yml
+  - DEPRECATED: Used for deployment of single node BigchainDB v1.3, SSL enabled for MongoDB authentication and authorization.
+- docker-compose.rdb.yml
+  - DEPRECATED: Used to deploy single node BigchainDB v1.3 with RethinkDB as the backend database.
+- docker-compose.benchmark.yml
+  - DEPRECATED: Used to deploy standalone BigchainDB v1.3 development environment.
+
+
+Majority of the above mentioned files are either not in use OR can be integrated in a way that all of them are being tested effectively with CI pipeline(s).
+
+We also have multiple Dockerfiles in the repository:
+
+- Dockerfile
+  - Dockerfile used to build and publish BigchainDB production images.
```

Comment:
any particular reason to keep `Dockerfile` and `Dockerfile-dev` separate?

Reply:
Yes, because we install BigchainDB using `pip install --no-cache-dir .` in `Dockerfile` and with `Dockerfile-dev` we will use `pip install --no-cache-dir -e .[dev]` so that changes are reflected at run time.